app/templates/gentelella/guest/event/_ticketing_box.html

```diff
@@ -270,9 +270,9 @@ <h3 class="panel-title">{{ _("Ticket Information") }}</h3>
         var discount_amount = -1;
         var promo_code_status = 0;
         var discount_code = "";
-        {% if code %}
+        {% if discount_code %}
```

Comment:
This works only for discount_code now. What about access_code?
Initially, check for "code" was for both access and discount.